painttiming.bs

```diff
@@ -117,8 +121,9 @@ An [=element=] |target| is <dfn export>contentful</dfn> when one or more of the
 * |target| is a [=replaced element=] representing an [=available=] [=image=].
 * |target| has a [=background-image=] which is a [=contentful image=], and its [=used=] [=background-size=] has non-zero width and height values.
 * |target| is a [=canvas=] with its [=context mode=] set to any value other than <code>none</code>.
+* |target| is a [=video element=] that [=represents=] its [=poster frame=] or the first video frame.
```

Comment:
This wording is not quite right, the `video element` can 'have' a `poster frame`, not represent one. And if it does not have one, we still need to specify that it becomes contentful when its first video frame is going to be displayed.

Reply:
Actually never mind, representing a poster frame is actually used in the HTML spec so I perhaps that's fine.

Reply:
I think we can keep the wording, but add something like 'and that frame is available'.